Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM php:7.4-fpm
+
+RUN apt-get update && apt-get install -y \
+      libxml2-dev \
+      libicu-dev \
+      libcurl3-dev \
+      libsqlite3-dev \
+      libedit-dev \
+      zlib1g-dev \
+      libfreetype6-dev \
+      libjpeg62-turbo-dev \
+      libmemcached-dev \
+      libzip-dev \
+      libonig-dev \
+      libpq-dev
+RUN docker-php-ext-install \
+        intl \
+        pdo_mysql \
+        pdo_pgsql \
+        opcache \
+        bcmath \
+        soap \
+        sockets \
+        zip \
+     && docker-php-ext-configure gd \
+     && docker-php-ext-install -j$(nproc) gd
+
+RUN pecl install redis \
+    && docker-php-ext-enable redis
```

Comment:
The docs still mention Redis, which I need to update (note it says "outdated"), but Redis isn't an available option for caching anymore. We only use APCu.